clusters.js

```diff
@@ -152,8 +152,10 @@ class DistanceMatrix {
         const clusters = elements.map(el => [el]);
 
         // Init matrix:
+        // eslint-disable-next-line prefer-const
```

Comment:
Oh, I also added the "prefer-const" rule, because it's a solid rule. The downside is that we seem to use `for (let foo of bar) {..}`. Not sure if we want to change those to "const" or keep as "let". I vaguely recall Firefox had issues w/ `for (const foo of bar) {...}` which is why we use `let`, depending on the target runtime.

TL;DR: I like `prefer-const`, so I enabled it globally, and explicitly disabled that error for each `for (let ...)` loop.